lib/manageiq/environment.rb

```diff
@@ -61,6 +61,7 @@ def self.install_bundler(root = APP_ROOT)
       system!("echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc") if ENV['CI']
       system!("gem install bundler -v '#{bundler_version}' --conservative")
       system!("bundle config path #{root.join('vendor/bundle').expand_path}", :chdir => root) if ENV["CI"]
+      system!("bundle config set force_ruby_platform true") if ENV["CI"]
```

Comment:
I think this is specifically for Travis, so can we change this to `ENV["TRAVIS"]` or something to that effect?  Reason I'm asking is because when we go to github actions this will probably no be needed.

Reply:
Maybe also with a comment that it was added for nokogiri 1.3

Reply:
Ok, updated